### DISCUSSION TITLE
What is the release schedule for Fish

### DISCUSSION THREAD

**Discussion by iacore** [fish-shell/fish-shell#10334] [Q&A] [answered]
Fish 3.7.0 introduces a bug that breaks glob expansion: https://github.com/fish-shell/fish-shell/issues/10205

I reported the bug on Jan 9. No new version was released with the bug fix yet. I wonder, does Fish have release engineering at all, or did the developer just forget about it.

**Comment by faho** [comment]
We have learned to not do release estimates.

Typically we don't do a *lot* of releases.

**Comment by zanchey** [comment, marked as answer]
Best effort basis.
